src/recipe/test/e2e/findRecipeById.controller.e2e-spec.ts

```diff
@@ -25,8 +28,10 @@ describe('Recipe Controller - Find By Id', () => {
 
   beforeAll(async () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [RecipeModule, AuthModule],
+      imports: [RecipeModule, AuthModule, AppModule],
     })
+      .overrideProvider(CACHE_MANAGER)
+      .useClass(CacheServiceMock)
```

Comment:
Please research if there is another way to test your code without having to mock the cache. Is this a "recommended" way of doing things? 

Maybe it would be a good idea to add a test case that would test the cache layer.

Please research the topic.